aws/src/main/resources/s3-signer-open-api.yaml

```diff
@@ -56,13 +56,18 @@ servers:
         description: Optional prefix to be appended to all routes
         default: ""
 
+tags:
+  - name: remote-signing
+    description: Requires server to support remote signing
```

Comment:
@danielcweeks, @nastra, @jackye1995, if we add some form of capabilities to express what a REST service supports, then maybe we should merge the signer spec into the main REST catalog spec. Then everything is in one place and options are clear.

Reply:
I agree, I think it's probably good to merge those 2 specs together

Reply:
I commented elsewhere. It makes sense to merge them to me, but let's not do it in this PR. We can take care of it in a follow up.